test/unit/math/rev/mat/fun/scale_matrix_exp_multiply_test.cpp

```diff
@@ -182,6 +182,14 @@ TEST(MathMatrix, scale_matrix_exp_multiply_vv) {
   test_scale_matrix_exp_multiply_vv(8, 2);
 }
 
+TEST(MathMatrix, matrix_exp_multiply_0x0) {
+  using stan::math::var;
+  const double t = 1.0;
+  Eigen::Matrix<var, -1, -1> A(0, 0);
+  Eigen::Matrix<var, -1, -1> B(0, 0);
```

Comment:
Can you add a test for A(0,1) and B(1,2)

Reply:
Added.